.github/workflows/pr-test-e2e.yml

```diff
@@ -68,7 +68,14 @@ jobs:
 
       - name: show pods in hosting cluster
         if: always()
-        run: kubectl --context kind-kubeflex get pods -A
+        run: |
+          date
+          kubectl --context kind-kubeflex get pods -A
+          kubectl --context kind-kubeflex get pods -A | grep -vw Running | grep -vw Completed | grep -v NAME | while read ns name rest; do echo; kubectl --context kind-kubeflex describe pod -n $ns $name; echo; kubectl --context kind-kubeflex logs -p -n $ns $name || true; echo; kubectl --context kind-kubeflex logs -n $ns $name || true; done
+
+      - name: show Deployment objects in hosting cluster
+        if: always()
+        run: kubectl --context kind-kubeflex get deployments -A
```

Comment:
Won't the logs of all the pods here make redundant the logs we do in subsequent sections, like ' show kubestellar controller manager log' ?

Reply:
Right now we have an overlap, which is better than a gap. Neither set covers the other.
We have some logging for every pod that is neither Running nor Completed.
We have some logging for a few pods regardless of their phase.